docs/resources.md

```diff
@@ -0,0 +1,97 @@
+# Zero components
+**s3 bucket: Terraform-state:** 
+`<project>-<env>-terraform-state`
+Terraform state s3 bucket is used for storing your terraform state in a terraform [remote backend][tf-remote-backend]
+
+**dynamo db: Terraform statelock**:
+Terraform uses [state lock][tf-remote-state] to ensure actions that could contradict each others do not happen at the same time 
+
+**IAM user: `<project>-ci-user`**:
+This user is given the ECR full-access policy, our intention is you will push your container images to ECR via this IAM
+
+**Secret manager: `ci-user-aws-key`**:
+This is a credential key pair generated when creating an IAM user, we then store it in secret manager.
```

Comment:
These credentials will be put into a CI system then used during the build and deploy process, most likely as environment variables.